### PR TITLE
Add self-seeding of data for this store, and Chec CLI config

### PR DESCRIPTION
**Pull request by ScopeyNZ**
This relies on the new `@chec/seeder` package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@chec/commerce.js": "^2.0.0-beta5",
+    "@chec/seeder": "^1.0.0-beta1",
```
Comment (robbieaverill) [inline package.json]: Dev only dependency maybe?

**Review by robbieaverill — APPROVED** [review]
Is the linking of product + asset handled by the chec-seed command internally, based on the fact that you've given a `link` attribute in the seed data?

File: package.json (hunk 1 above)
Comment (ScopeyNZ) [reply package.json]: Yeah. I wasn't sure. All the react-script stuff is a dependency... I guess you still need that to build a production bundle though...

**Comment by ScopeyNZ** [comment]
Yeah the product/asset stuff is pretty custom for products/assets. The link thing works, but only for that relationship. Could be possibility to make that a little more generic going forwards...

File: package.json (hunk 1 above)
Comment (robbieaverill) [reply package.json]: Our "production" for this repo is GitHub pages, which won't need this because it has a pre-defined API key right?

Comment (ScopeyNZ) [reply package.json]: I think we can sort of think of this repo as not actually having a production branch. There's really 3 repos in one here. I'll move the dependency.

**Pull request merged**